### PR TITLE
Add config for HelixTaskWorkflowMangager

### PR DESCRIPTION
**Pull request by snalli**
Add config for HelixTaskWorkflowMangager

### CODE REVIEW

File: config/HelixTaskWorkflowManagerConfig.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+
+zk.layout.file.path=../config/zkLayout.json
+cluster.name=Ambry-corp
+# workflow.name=aggregationRecurrent
+# workflow.name=mysqlPartitionClassTriggerRecurrent
+workflow.name=mysqlTriggerRecurrent
+stats.reports.to.aggregate=PARTITION_CLASS_REPORT,ACCOUNT_REPORT
+delete.specified.workflow=false
+recurrent.interval.in.minutes=10800
```
Comment (ankagrawal) [inline config/HelixTaskWorkflowManagerConfig.properties]: nit: add new line at the end of file.

Comment (ankagrawal) [reply config/HelixTaskWorkflowManagerConfig.properties]: Also why are we adding linkedin specific config file in open source code?

Comment (snalli) [reply config/HelixTaskWorkflowManagerConfig.properties]: added newline but it doesn't show up, well actually it does. Without it , there'd be a red marker.

File: config/HelixTaskWorkflowManagerConfig.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+zk.layout.file.path=../config/zkLayout.json
+cluster.name=<cluster name>
+workflow.name=<workflow name>
+stats.reports.to.aggregate=PARTITION_CLASS_REPORT,ACCOUNT_REPORT
+delete.specified.workflow=false
+recurrent.interval.in.minutes=10800
```
Comment (justinlin-linkedin) [inline config/HelixTaskWorkflowManagerConfig.properties:18]: Change this to 10, now that the aggregation task should be executed every 10 minutes

Comment (snalli) [reply config/HelixTaskWorkflowManagerConfig.properties:18]: done but this is just sample config

File: config/HelixTaskWorkflowManagerConfig.properties (hunk 1 above)
Comment (snalli) [reply config/HelixTaskWorkflowManagerConfig.properties]: nothing in this pr is linkedin specifc atm

**Review by justinlin-linkedin — APPROVED** [review]

**Pull request merged**